### ISSUE TITLE
Consider Adding Hudson Rock Cybercrime Intelligence Free Integration

### ISSUE THREAD

**Issue by hudsonrock-partnerships** [Alfredredbird/tookie-osint#55]
Consider adding Hudson Rock's complimentary data to receive additional intelligence about a username that was compromised in global Infostealer attacks.

Username search sample: https://cavalier.hudsonrock.com/api/json/v2/osint-tools/search-by-username?username=tookie

Thank you.

**Comment by Alfredredbird** [comment]
That is very interesting. Could you provide more details?

**Comment by hudsonrock-partnerships** [comment]
Sure. Hudson Rock is a cybercrime intelligence company. We specialize in obtaining data from machines infected by the Infostealer malware. The compromised data includes usually login URLs, email addresses/**usernames**, passwords, and other sensitive data. 

As part of our contribution to the OSINT and CTI communities, we opened a small part of our data for integration with other projects. We already successfully integrated with projects like [UserFinder](https://github.com/mishakorzik/UserFinder) and [WhatsMyName](https://github.com/WebBreacher/WhatsMyName/) 

We are fully aware that you are a social-media-oriented tool, but sometimes additional intelligence from another source can help the end user. 

We have different endpoints that are open for integration. However, the username will be the most relevant for your case.
